docs/swagger.yaml

```diff
@@ -179,8 +179,12 @@ paths:
                   bottomLine:
                     currency: NOK
                     tipAmount: 0
-                    giftCardAmount: 0
                     terminalId: vipps_pos_122
+                    paymentSource:
```

Comment:
In ecom common this is called paymentSource**s**
Can you update the swagger to be with the "s" as well?